README.md

```diff
@@ -34,25 +34,25 @@ Developer documentation
 
 ### How to set the environment
 
-* Install NPM (v8 or higher) / Node.js (v6 or higher)
-* npm install -g tfx-cli
-* npm run install-dep-full
+* Install NPM (v8 or higher) / Node.js (v14 or higher)
+* `npm install -g tfx-cli`
+* `npm run install-dep-full`
 
 ### Package a production build
 
-* npm run build
+* `npm run build`
 
 ### Package a test build
 
-* npm run test-build -- --publisher <publisher-id>
+* `npm run test-build -- --publisher <publisher-id>`
```

Comment:
`<publisher-id>` was hidden from the rendered markdown because GitHub was assuming this is an HTML tag, using inner code syntax fixes the issue